sections/05_treat.md

```diff
@@ -11,6 +11,148 @@ other interventions?  As an example, [@doi:10.1016/j.jalz.2015.01.010]
 predicts individuals who are most likely to decline during a clinical trial
 and benefit from the treatment.*
 
+There has been sustained interest in applying deep learning to clinical
+decision making for over two decades. In 1996, Tu
+[@doi:10.1016/S0895-4356(96)00002-9] compared the effectiveness of artificial
+neural networks and logistic regression, questioning whether deep learning
+would replace traditional statistical methods for predicting medical
+outcomes. He posited that while neural networks have several advantages in
```

Comment:
"medical outcomes" is broad.  Maybe "medical outcomes, such as..." and give an example or two of what was discussed in this paper?